hardware/raspberrypi/bootmodes/bootflow_2711.md

```diff
@@ -47,12 +48,11 @@ Please see the [bootloader configuration](../bcm2711_bootloader_config.md) page
    * else if boot-mode == `SD CARD`
       * Attempt to load firmware from the SD card
          * Success - run the firmware
-         * Failure - continue
-   * else if boot-mode == `NETWORK`
-      * If boot-mode == `NETWORK` then 
-         * Use DHCP protocol to request IP address
-         * Load firmware from the DHCP or statically defined TFTP server
-         * If the firmware is not found or a timeout or network error occurs then continue
+         * Failure - continue   
+   * If boot-mode == `NETWORK` then 
```

Comment:
I suspect this still ought to be "else if" ?

Reply:
good point I trimmed the wrong line!